src/data/services/DiscoveryDataApiService.js

```diff
@@ -10,6 +11,7 @@ class DiscoveryDataApiService {
     const queryParams = {
       editable: 1,
       exclude_utm: 1,
+      include_restricted: restrictionTypeValues,
```

Comment:
hmmm, do we not need any test updates for this?

Reply:
No. There are virtually no tests for this file.